app/Rules/Uppercase.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Rules;
+
+use Closure;
+use Illuminate\Contracts\Validation\ValidationRule;
+use Illuminate\Support\Facades\Log;
+
+class Uppercase implements ValidationRule
+{
+    /**
+     * Run the validation rule.
+     *
+     * @param  \Closure(string): \Illuminate\Translation\PotentiallyTranslatedString  $fail
+     */
+    public function validate(string $attribute, mixed $value, Closure $fail): void
+    {
+        if (ucfirst($value) !== substr($value,0)) {
```

Comment:
This code is not working correctly. In the second condition, you should only use `$value.`

`if (ucfirst($value) !== $value) { }`

Reply:
Thanks for noticing Kadir
yes, I think that does not make sense to use `substr($value, 0)` without the length param,
but if you check `substr($value,0)` it will return the same `$value` because of that the test passed.